### PR TITLE
Disable failed tests for 9 issues.

### PR DESCRIPTION
**Pull request by Jiayili1**
Disabled following issues:
#19462 
#19458 
#19456 
#19455 
#18806 
#19384 
#19383 
#19286 
#19179 

cc: @danmosemsft @safern

### CODE REVIEW

File: src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs [hunk 1]
```diff
@@ -149,6 +149,7 @@ public void TestReflectInitializationViaSubscription3()
         /// Test request cancellation: reflection hook does not throw
         /// </summary>
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19286")]
```
Comment (safern) [inline src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs]: This tests seems to be fixed by @lmolkova but I'm not sure as we didn't get a response from her when we asked on the issue. 

So I don't know if we should disable this test. Also there is two issues tracking this test, so could you please update the issue # to 19280?

@danmosemsft should we disable this test? or wait until we get a response from @lmolkova?

File: src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs [hunk 2]
```diff
@@ -44,6 +44,7 @@ public void Dispose()
 
         [Theory]
         [ClassData(typeof(SslProtocolSupport.UnsupportedSslProtocolsTestData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19458")]
```
Comment (stephentoub) [inline src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs]: This should be reverted.  It was addressed by https://github.com/dotnet/corefx/pull/19477.

File: src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs [hunk 3]
```diff
@@ -74,6 +75,7 @@ public void Dispose()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19456")]
```
Comment (stephentoub) [inline src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs]: This should be reverted.  It was addressed by https://github.com/dotnet/corefx/pull/19477.

File: src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs (hunk 2 above)
Comment (safern) [inline src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs]: I'll revert it.

File: src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs (hunk 3 above)
Comment (safern) [inline src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs]: I'll revert it.

File: src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs (hunk 1 above)
Comment (lmolkova) [inline src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs]: Please re-enable the test, it should be fixed with #19333 and #19280 is updated.

Comment (safern) [inline src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs]: Thanks for the update. Re-enabling.

File: src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs (hunk 3 above)
Comment (safern) [inline src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs]: Fixed.

File: src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs (hunk 2 above)
Comment (safern) [inline src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs]: Fixed.

File: src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs (hunk 1 above)
Comment (safern) [inline src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs]: Fixed.

**Pull request merged**